oe_theme.theme

```diff
@@ -1522,3 +1523,84 @@ function oe_theme_preprocess_page(array &$variables): void {
     'id' => ['main-content'],
   ]);
 }
+
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function oe_theme_preprocess_oe_organisation(&$variables): void {
```

Comment:
TODO: move it to oe_theme_content_entity_organisation module when it will be done in OPENEUROPA-2626.